tests/conftest.py

```diff
@@ -15,6 +16,24 @@
 import mesonpy
 
 
+def adjust_packaging_platform_tag(platform: str) -> str:
```

Comment:
Since `packaging` is no longer used, I assume the `manylinux` part of this is also no longer needed?

The change seems right to me; neither `meson-python` nor `packaging` can know whether or not the wheel being produced is manylinux compliant, and there should be no attempt to deal with that at all (that's for `auditwheel`).

Reply:
This is only used in the tests to compare what `meson-python` generates with what `packaging` generates. `packaging` generates the very specific `manylinux` platform tag while `meson-python` does not. Thus we need to adapt the former version of the tag to the latter. However, while I was writing this I realized that this does not handle `muslinux` tags, thus the tests are broken on platforms using that libc. Fixing it.

Reply:
Ah okay, got it. So we use `packaging` in the test suite to ensure that there's no silent mismatches that result in wheels we produce being uninstallable - that seems quite useful.